src/gui/widgets/multiline_text.hpp

```diff
@@ -143,6 +143,19 @@ class multiline_text : public text_box_base
 		update_layout();
 	}
 
+	void set_use_markup(bool use_markup) override
+	{
+		text_box_base::set_use_markup(use_markup);
```

Comment:
I think you can use `using text_box_base::set_use_markup;`

Reply:
removed function.

Reply:
Still seeing this one? Looks like it snuck through.

Reply:
removed.